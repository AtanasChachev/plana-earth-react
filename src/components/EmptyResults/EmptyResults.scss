.empty-results {
  max-width: 440px;
  margin: 0 auto;
  padding: 20px;
  text-align: center;

  &__image {
    max-width: 200px;
    height: auto;
  }

  &__title {
    margin: 10px 0;
  }

  &__description {
    font-size: 16px;
    line-height: 1.5;
  }
}