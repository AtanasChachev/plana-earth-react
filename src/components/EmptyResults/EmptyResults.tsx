import search from 'assets/images/search.png';

import './EmptyResults.scss';

const EmptyResults = (): JSX.Element => {
  return (
    <div className="empty-results">
      <img className="empty-results__image" src={search} alt="Search Icon" />

      <h2 className="empty-results__title">No Results Found</h2>
      <p className="empty-results__description">We were not able to find results by the given criteria. Please try to change the filters in the left column.</p>
    </div>
  );
};

export { EmptyResults };