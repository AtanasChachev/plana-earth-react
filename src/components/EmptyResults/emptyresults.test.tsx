import { render } from '@testing-library/react';
import { EmptyResults } from './EmptyResults';

const setup = () => {
  const { asFragment } = render(<EmptyResults />);

  return {
    asFragment,
  };
};

describe('<EmptyResults />', () => {
  test('<EmptyResults /> is rendering correctly', () => {
    const { asFragment } = setup();

    expect(asFragment()).toMatchSnapshot();
  });
});