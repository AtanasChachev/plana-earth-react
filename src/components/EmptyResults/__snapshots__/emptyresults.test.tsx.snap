// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyResults /> <EmptyResults /> is rendering correctly 1`] = `
<DocumentFragment>
  <div
    class="empty-results"
  >
    <img
      alt="Search Icon"
      class="empty-results__image"
      src="search.png"
    />
    <h2
      class="empty-results__title"
    >
      No Results Found
    </h2>
    <p
      class="empty-results__description"
    >
      We were not able to find results by the given criteria. Please try to change the filters in the left column.
    </p>
  </div>
</DocumentFragment>
`;
